packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -280,8 +289,16 @@ export class AppManagementClient implements DeveloperPlatformClient {
       )
   }
 
-  async templateSpecifications(_appId: string): Promise<ExtensionTemplate[]> {
-    return stubbedExtensionTemplates()
+  async templateSpecifications({organizationId}: MinimalAppIdentifiers): Promise<ExtensionTemplate[]> {
+    const allExtensions = await Promise.all(templateJsonUrls.map(async (url) => {
+      const response = await fetch(url)
+      return response.json() as Promise<GatedExtensionTemplate[]>
+    })).then(templates2DArray => templates2DArray.flat(1))
+    const allBetaFlags = Array.from(new Set(allExtensions.map(ext => ext.organizationBetaFlags ?? []).flat()))
+    const enabledBetaFlags = await this.organizationBetaFlags(organizationId, allBetaFlags)
```

Comment:
Are we going to check the CLI version in BP as well? Because we will still need to filter templates by that, right?